### DISCUSSION TITLE
The developer should be able to implement CRUD operations using some components

### DISCUSSION THREAD

**Discussion by malsioufi** [appbaseio/reactivesearch#1564] [Q&A]
**Affected Projects**
React

**Is your feature request related to a problem? Please describe.**
Usually when we build search solution we like to be able to modify, delete or create new contents.

**Describe the solution you'd like**
Components that allows the user to add/ modify/ remove data, similar to what we have in Dejavu but to be able to do it in my own react application.

**Describe alternatives you've considered**
N/A

**Additional context**

**Comment by siddharthlatest** [comment]
Since create/edit/update components (typically form kind of interfaces) have no dependency on the search state, you can build those with any JS UI components library and make a fetch/ajax request to update the search index.

**Reply by malsioufi** [reply]
That's right for end users of the application, but for the content creators, I think, usually they search using some criteria, then they try for example to update something or delete something. Just an opinion..
